src/assigner.ts

```diff
@@ -1,14 +1,94 @@
 /**
  * Assigner
- * 
- * @module bot.ts
- * @author Daniel Dimitrov <danieldimitrov2304@gmail.com>
+ *
+ * @module assigner.ts
+ * @author Daniel Batanov <batanoff.s@protonmail.com>
  * @description This module is responsible for assigning crawl tasks for the bot.
  */
 
 // Import dependencies
+import prisma from "./db/prisma/prisma";
+import logger from "./utils/logger";
+
+// Declare and define constant for crawl task status.
+// Better to move it but where to? Suggestion db/prisma/constants - file constants.ts
+const CRAWL_TASK_STATUS = {
+    PENDING: "PENDING",
+    IN_PROGRESS: "IN_PROGRESS",
+    COMPLETED: "COMPLETED",
+    FAILED: "FAILED",
+};
```

Comment:
Import this 'import { CrawlTaskStatus } from "@prisma/client"'
This is the enum type from the schema. From there you can access the statuses directly.
No need for this.